docs/docs/data-structures/tree/schema-evolution/allowed-types-rollout.mdx

```diff
@@ -54,14 +54,9 @@ const configB = new TreeViewConfiguration({
     schema: schemaB,
 });
 const viewB = treeB.viewWith(configB);
-// check that we can read the tree
-assert.deepEqual(viewB.root, 5);
-// upgrade to schema B: this is a no-op
-viewB.upgradeSchema();
-synchronizeTrees();
```

Comment:
I removed the no-op call to `upgradeSchema` because this guide is supposed to show app authors what to do and calling `upgradeSchema` is not something they need to do. The existing comment below about how it is unnecessary seems sufficient.